macro-pdfviewer-api/src/main/java/com/xwiki/pdfviewer/internal/macro/PDFViewerMacro.java

```diff
@@ -48,29 +65,56 @@
 @Singleton
 public class PDFViewerMacro extends AbstractMacro<PDFViewerMacroParameters>
 {
+    private final List<String> delegatedRightsValues = List.of("1", "true", "yes");
+
+    @Inject
+    private AuthorizationManager authorizationManager;
+
     @Inject
     private TemplateManager templateManager;
 
+    @Inject
+    @Named("resource/standardURL")
+    private EntityReferenceResolver<String> urlEntityReferenceResolver;
+
+    @Inject
+    @Named("current")
+    private EntityReferenceResolver<String> entityReferenceResolver;
+
     @Inject
     private ScriptContextManager scriptContextManager;
 
+    @Inject
+    private Provider<XWikiContext> wikiContextProvider;
+
+    @Inject
+    private Provider<MJSFilter> mjsFilterProvider;
+
     /**
      * Create and initialize the descriptor of the macro.
      */
     public PDFViewerMacro()
     {
         super("PDF Viewer", "View PDF attachments inside wiki pages without downloading or importing them.",
             PDFViewerMacroParameters.class);
+        // Error when building the project on 14.10.
+//        setDefaultCategory(DEFAULT_CATEGORY_CONTENT);
```

Comment:
Leftover?

Reply:
I'm getting this error in 14.10, so I commented the line for the moment: 

> Cannot resolve method 'setDefaultCategory' in 'PDFViewerMacro'

Reply:
Yes, because of https://github.com/xwiki/xwiki-rendering/commit/40e5d9534e9c5af62ef27f7d2bc8ca9ca3539e8f#diff-c291fae6ef5c3cd4ddb2d348a1fb6fa6b4d98cb7d4c910319fa1ea0d954b830cR248 . This method has been replaced.